src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/DefaultDocumentResolverTest.cs

```diff
@@ -99,23 +96,5 @@ public void TryResolveDocument_AsksPotentialParentProjectForDocumentItsNotTracki
             Assert.True(result);
             Assert.Same(expectedDocument, document);
         }
-
-        [Fact]
-        public void TryResolveDocument_AsksMiscellaneousProjectForDocumentItIsNotTracking_ReturnsFalse()
```

Comment:
Made redundant by https://github.com/dotnet/aspnetcore-tooling/pull/2298/files#diff-7761c270789b837eaeeb4c0242790d58R57 as we no longer to a second, manual check with the MISC project. We rely on the project resolver to check both "potential" and misc projects.